ingestion/src/metadata/profiler/interface/profiler_protocol.py

```diff
@@ -128,21 +128,19 @@ def get_profile_sample_config(
                 profile_sample=entity_config.profileSample,
                 profile_sample_type=entity_config.profileSampleType,
             )
-        if hasattr(entity.tableProfilerConfig, "profileSample") and hasattr(
-            entity.tableProfilerConfig, "profileSampleType"
-        ):
-            return ProfileSampleConfig(
-                profile_sample=entity.tableProfilerConfig.profileSample,
-                profile_sample_type=entity.tableProfilerConfig.profileSampleType,
-            )
-
         if source_config.profileSample:
             return ProfileSampleConfig(
                 profile_sample=source_config.profileSample,
                 profile_sample_type=source_config.profileSampleType,
             )
-
-        return None
+        try:
+            return ProfileSampleConfig(
+                profile_sample=entity.tableProfilerConfig.profileSample,
+                profile_sample_type=entity.tableProfilerConfig.profileSampleType,
+            )
+        except AttributeError:
+            # Exception Triggered if profileSample attribute not available in tableProfilerConfig
+            return None
```

Comment:
what is that? I believed we had it this way in `main`

```python
    @staticmethod
    def get_profile_sample_config(
        entity: Table,
        entity_config: Optional[TableConfig],
        source_config: DatabaseServiceProfilerPipeline,
    ) -> Optional[ProfileSampleConfig]:
        """_summary_

        Args:
            entity (Table): table entity object
            entity_config (Optional[TableConfig]): table config object from yaml/json file
        Returns:
            Optional[dict]: dict
        """
        if entity_config:
            return ProfileSampleConfig(
                profile_sample=entity_config.profileSample,
                profile_sample_type=entity_config.profileSampleType,
            )

        try:
            profile_sample = entity.tableProfilerConfig.profileSample
        except AttributeError:
            pass
        else:
            if profile_sample:
                return ProfileSampleConfig(
                    profile_sample=entity.tableProfilerConfig.profileSample,
                    profile_sample_type=entity.tableProfilerConfig.profileSampleType,
                )

        if source_config.profileSample:
            return ProfileSampleConfig(
                profile_sample=source_config.profileSample,
                profile_sample_type=source_config.profileSampleType,
            )

        return None
```

Reply:
ohh, if it's already taken care of, let me revert that back